### ISSUE TITLE
Option to hide new tab button at the end of the list (+)

### ISSUE THREAD

**Issue by DoTheEvolution** [piroor/treestyletab#579]
TL;DR
option to turn off / hide this?
http://i.imgur.com/eQP7Bx2.png

---

I am probably switching from opera to firefox and Tree-style-tabs are important in this transition since I am accustom to tabs on the right side.
They are superior in preferences and features compared to the native tabs in opera, but I really dislike the new tab button at the end of the list.
1. its not a static position, it moves around which is against my personal preference
2. the whole area under the tabs is the new tab button if you click it with the wheel or double left click, so to me it feels like unnecessary additional clutter that obscure my easy fast eye recognition of which tab is the last

**Comment by mxmilkb** [comment]
Maybe the text on the help page has changed, and/or maybe the method to hide the new tab button has changed, or maybe I'm getting this wrong, but I'm not seeing a way to remove the new tab button from the tst sidebar by using the Customize method. The sidebar disappears in that mode. I have tried disabling the chrome css that hides the top tabs and sidebar header, but to no change. Might there be something I am missing? Thanks!

**Comment by irvinm** [comment]
Have you tried this?  https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-the-new-tab-button-at-the-bottom-edge-of-the-tab-bar-1591

**Comment by mxmilkb** [comment]
I hadn't before, but I have now and I still see the new tab button. Hmm.

**Comment by irvinm** [comment]
Hmm ... I just tried it and it worked for me.  Did you place this in the TST "extra style rules" section?

```
.newtab-button-box {
  display: none;
}
#tabbar {
  bottom: 0 !important; /* Eliminate dead space on bottom */
}
```

**Comment by mxmilkb** [comment]
Bingo, silly me, I was putting it in my profile chrome css. Problem now resolved, thanks :)
